docs/content.zh/docs/ops/metrics.md

```diff
@@ -1428,6 +1428,16 @@ Note that the metrics are only available via reporters.
       <td>Maximum recorded duration of a single consecutive period of the task being in the hard back pressure state in the last sampling period. Please check softBackPressuredTimeMsPerSecond and hardBackPressuredTimeMsPerSecond for more information.</td>
       <td>Gauge</td>
     </tr>
+    <tr>
```

Comment:
what about the mailbox queue length? Should we add it as well?

Reply:
That would see it as meaningful. I didn't want to extend the scope too much as the issue's scope is already extended with the throughput but I would be +1 for it. Although I need to check if this feasible in an easy way. Counting (which methods need to inc and dec as it's not fully trivial with the createBatch semantics) vs. size method (what interval? how to do that with a metrics class? also note that size() is marked @VisibleForTesting and acquires a lock).

What do you think should we invest the effort now?